docker-compose.yml

```diff
@@ -7,7 +7,7 @@ services:
       - "${UI_PORT}:5122"
     volumes:
       - "${WORK_DIR}/data:/siglens/data"
-      - "${WORK_DIR}/siglens.db:/siglens/siglens.db"
+      - "${WORK_DIR}/data/siglens.db:/siglens/data/siglens.db"
```

Comment:
nit: I don't think we need to mount the `siglens.db` volume separately like this now. Since it is already inside `data` folder.